web/src/lib/components/shared-components/settings/setting-textarea-wysiwyg.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
```

Comment:
Can you help updating this file with Svelte 5 syntax as well?

Reply:
Of course, updated the file